README.md

```diff
@@ -23,7 +24,7 @@ But you won't need to worry about installing any of these - they are automatical
 
 ## How to run
```

Comment:
mmm, if we're really keeping this, we should remind the user of the need to have a running instance of postgres, should we not?

Reply:
Also, a note of warning due to apigw mediation might come in handy - when using the service from a dojot instance, the user should never be exposed to internal service addresses and ports. All examples assume the "standalone" execution of the service, and the user might feel lost to the changes in baseurl address (tcp port, more precisely) between apib and this.

Reply:
Ok! I'll add a note related to postgres and add a section (or something alike) indicating how the user should send the requests using a full dojot setup.